### PR TITLE
patch quantization

### PR DESCRIPTION
**Pull request by Slyne**
Hi @titu1994 
I've further tested the quantization part code and I get the below results:

First, calibrate the model:
```bash
python speech_to_text_calibrate.py --asr_model=./QuartzNet15x5-Zh-NR.nemo  --dataset=/raid/data/data_aishell2/dev_manifest.json --dont_normalize_text
```
Then, test the quant model:
```bash
python3 speech_to_text_quant_infer.py --asr_model=./QuartzNet15x5-Zh-NR-max-256.nemo --dont_normalize_text --use_cer --onnx --dataset=/raid/data/data_aishell2/dev_manifest.json
```
I get a CER of  7.34%.

Finally, to test with trt with the generated onnx model:
```bash
python3 speech_to_text_quant_infer_trt.py --asr_model=./QuartzNet15x5-Zh-NR.nemo  --asr_onnx=./QuartzNet15x5-Zh-NR-max-256.onnx --dont_normalize_text --use_cer --qat --dataset=/raid/data/data_aishell2/dev_manifest.json --batch_size=256
```
I get a CER of 9.08% with batch_size=256,  8.06% with batch_size=256 & sort dev set by audio length,  7.95% with batch_size=1.

I know it is normal to have 0.01%~0.05% error tolerance due to the padding of Quartznet. But the above result shows that there might be some issues with the script or the generated trt engine file.

(The size of the generated engine file is 24MB, which seems reasonable and the original model trt engine file is 95MB.)

I build my environment by starting from `nvcr.io/nvidia/pytorch:21.05-py3` + tensorrt 8 EA + nemo_toolkit[all].

Not sure if I made any mistakes in the above steps. Please correct me.

### CODE REVIEW

**Review by titu1994 — APPROVED** [review]
Looks correct

**Comment by Slyne** [comment]
@titu1994 Could you help check the above issue ?

**Comment by titu1994** [comment]
I'm not particularly familiar with the quantization aware training methods. Will request someone from team to help with this.
